lib/util/git/index.ts

```diff
@@ -572,8 +579,8 @@ export async function getFileList(): Promise<string[]> {
     .map((line) => line.split(regEx(/\t/)).pop()!);
 }
 
-export function getBranchList(): string[] {
-  return Object.keys(config.branchCommits);
+export function getBranchList(): LongCommitSha[] {
+  return Object.keys(config.branchCommits) as LongCommitSha[];
 }
```

Comment:
Doesn't this return branch names and not commits?

Reply:
you are right, the keys are the names, the values are the commit hashes (and these are of the right type)